src/extension.ts

```diff
@@ -288,15 +293,10 @@ export async function _runAndDisplayDfa(context:vscode.ExtensionContext ,runInfo
 	}
 }
 
-export async function _shouldProceedWithDfaRun(context: vscode.ExtensionContext, channel: vscode.LogOutputChannel): Promise<boolean> {
+export async function _shouldProceedWithDfaRun(context: vscode.ExtensionContext): Promise<boolean> {
 	if (context.workspaceState.get(Constants.WORKSPACE_DFA_PROCESS)) {
-		await vscode.window.showInformationMessage(messages.graphEngine.stopDfaRunConfirmationText, messages.graphEngine.stopDfaRunConfirmationYes, messages.graphEngine.stopDfaRunConfirmationNo)
-			.then(async answer => {
-				if (answer === messages.graphEngine.stopDfaRunConfirmationYes) {
-					await _stopExistingDfaRun(context, channel);
-				}
-			});
-			return false;
+		await vscode.window.showInformationMessage(messages.graphEngine.existingDfaRunText);
```

Comment:
I am no longer giving an direct option to cancel the previous run when subsequent runs are triggered. The user has to click on the statusbar and cancel the previous run. During testing I found allowing cancellation from multiple places resulted in unpredictable behavior sometimes. Hence this change.